docs/next/guides/upgrade-and-migration/migrating-from-9.0-to-10.0.md

```diff
@@ -0,0 +1,70 @@
+| `beforeRender` | `beforeViewRender` |
+| `afterRender`  | `afterViewRender`  |
+
+There are still hooks that are named `beforeRender` and `afterRender`, but they do completely new things now. For more details, see the [Handsontable 10.0 API reference](@/api/hooks.md):
+- [`beforeRender`](@/api/hooks.md#beforerender)
+- [`afterRender`](@/api/hooks.md#afterrender)
+- [`beforeViewRender`](@/api/hooks.md#beforeviewrender)
+- [`afterViewRender`](@/api/hooks.md#afterviewrender)
+
+## Step 2: Adapt to the HyperFormula dependency update
+
+In Handsontable 10.0.0, we updated the optional HyperFormula dependency from `0.6.2` to `^1.1.0` ([#8669](https://github.com/handsontable/handsontable/pull/8669)).
+
+For more details on the breaking changes between HyperFormula 0.6.x and HyperFormula 1.0.x, see the [migration guide](https://handsontable.github.io/hyperformula/guide/migration-from-0.6-to-1.0.html).
+
+## Step 3: Adapt to the configuration options' new default values
+
+In Handsontable 10.0.0, we changed the default values of the [`autoWrapCol`](@/api/options.md#autowrapcol) and [`autoWrapRow`](@/api/options.md#autowraprow) [configuration options](@/guides/getting-started/setting-options.md), from `true` to `false` ([#8662](https://github.com/handsontable/handsontable/pull/8662)):
+
+| Before                                              | After                                                |
+| --------------------------------------------------- | ---------------------------------------------------- |
+| [`autoWrapCol: true`](@/api/options.md#autowrapcol) | [`autoWrapCol: false`](@/api/options.md#autowrapcol) |
+| [`autoWrapRow: true`](@/api/options.md#autowraprow) | [`autoWrapRow: false`](@/api/options.md#autowraprow) |
+
+We also changed the default values for the [`rowsLimit`](@/api/copypaste.md#rowslimit) and [`columnsLimit`](@/api/copypaste.md#columnslimit) options of the [`CopyPaste`](@/api/copypaste.md) plugin, from `1000` to `Infinity` ([#8676](https://github.com/handsontable/handsontable/pull/8676)):
+
+| Before                                                  | After                                                       |
+| ------------------------------------------------------- | ----------------------------------------------------------- |
+| [`rowsLimit: 1000`](@/api/copypaste.md#rowslimit)       | [`rowsLimit: Infinity`](@/api/copypaste.md#rowslimit)       |
+| [`columnsLimit: 1000`](@/api/copypaste.md#columnslimit) | [`columnsLimit: Infinity`](@/api/copypaste.md#columnslimit) |
+
+## Step 4: Adapt to the hook changes
+
+In Handsontable 10.0, we unified the naming of an argument used by the [`beforeOnCellMouseDown`](@/api/hooks.md#beforeoncellmousedown) and [`beforeOnCellMouseOver`](@/api/hooks.md#beforeoncellmouseover) hooks ([#8591](https://github.com/handsontable/handsontable/pull/8591)):
+
+| Before              | After        |
+| ------------------- | ------------ |
+| `blockCalculations` | `controller` |
```

Comment:
This PR not only unified the naming of the argument, [but also the `cells`/`cell` property of the object behind it](https://github.com/handsontable/handsontable/pull/8591/files#diff-ed84d7ae87be0139136a8e07547d633af52a2c1c30d3b4ade974f5822df881ebR1977), and it's not explained in the PR description itself.

Could we maybe make this table entry a link to the docs for the hook and mention the change in the "Step 4" description?

Reply:
@jansiegel - right, how about this? 995dafc

Reply:
@kirszenbaum Magnificent 👌